corehq/apps/es/utils.py

```diff
@@ -193,3 +193,13 @@ def get_es_reindex_setting_value(name, default):
     if ES_REINDEX_LOG[-1] != getattr(settings, 'ES_MULTIPLEX_TO_VERSION', None):
         return default
     return getattr(settings, name, default)
+
+
+def get_user_domain_memberships(user):
```

Comment:
is this just 

```
def get_user_domain_memberships(user):
    if user['doc_type'] == 'CommCareUser':
        return user['domain_membership']
    elif user['doc_type'] == 'WebUser':
        return user['domain_memberships']:
    ```

Reply:
yep, except the first `return` needs `[]` around it.  I updated it